clarity_epp/export/tecan.py

```diff
@@ -13,9 +13,9 @@ def samplesheet(lims, process_id, output_file):
 
     for placement, artifact in process.output_containers()[0].placements.iteritems():
         placement = ''.join(placement.split(':'))
-        well_plate[placement] = artifact.name
+        well_plate[placement] = artifact.name.split('_')[0]
```

Comment:
this is the only place you split the artifact.name, could you elaborate?

Reply:
Discussed in teams.